scripts/scr_initialize_custom/scr_initialize_custom.gml

```diff
@@ -1,1352 +1,1338 @@
-	    if (heh=12) then global.founding_secret="Death Guard";
-	    if (heh=13) then global.founding_secret="Thousand Sons";
-	    if (heh=14) then global.founding_secret="Black Legion";
-	    if (heh=15) then global.founding_secret="Word Bearers";
-	    if (heh=16) then global.founding_secret="Salamanders";
-	    if (heh=17) then global.founding_secret="Raven Guard";
-	}
-
-
-
-
-	company_title[0]="";
-	var i;i=0;repeat(40){i+=1;company_title[i]=obj_creation.company_title[i];}
-
-
-
-
-
-	home_name=obj_creation.homeworld_name;obj_creation.restart_home_name=home_name;
-	chapter_name=obj_creation.chapter;
-	// fortress_name="";
-	flagship_name=obj_creation.flagship_name;obj_creation.restart_flagship_name=flagship_name;
-	sector_name=scr_sector_name();
-	icon=obj_creation.icon;
-	icon_name=obj_creation.icon_name;
-	man_size=0;
-	psy_powers=obj_creation.discipline;
-
-	strin=string(obj_creation.adv[1])+string(obj_creation.adv[2])+string(obj_creation.adv[3])+string(obj_creation.adv[4]);
-	strin2=string(obj_creation.dis[1])+string(obj_creation.dis[2])+string(obj_creation.dis[3])+string(obj_creation.dis[4]);
-
-	progenitor_disposition=obj_creation.disposition[1];
-	astartes_disposition=obj_creation.disposition[6];
-	imperium_disposition=obj_creation.disposition[2];
-	guard_disposition=obj_creation.disposition[2];
-	inquisition_disposition=obj_creation.disposition[4];
-	ecclesiarchy_disposition=obj_creation.disposition[5];
-	mechanicus_disposition=obj_creation.disposition[3];
-	other1_disposition=0;
-	other1="";
-
-	tolerant=0;var o;o=0;repeat(4){o+=1;if (obj_creation.dis[o]="Tolerant") then tolerant=1;adv[o]=obj_creation.adv[o];dis[o]=obj_creation.dis[o];}
-
-	battle_barges=0;
-	strike_cruisers=0;
-	gladius=0;hunters=0;
-
-	recruiting_type=obj_creation.recruiting;
-	aspirant_trial=obj_creation.aspirant_trial;
-	recruiting_name=obj_creation.recruiting_name;
-	home_type=obj_creation.homeworld;
-	home_name=obj_creation.homeworld_name;
-	fleet_type=obj_creation.fleet_type;
-
-
-
-	// if not custom
-
-
-	if (obj_creation.fleet_type!=1){
-	    battle_barges=1;
-	    strike_cruisers=6;
-	    gladius=7;
-	    hunters=3;
-	    // obj_controller.fleet_type="Fleet";
-	}
-	if (obj_creation.fleet_type=1){
-	    strike_cruisers=8;
-	    gladius=7;
-	    hunters=3;
-	    // obj_controller.fleet_type="Homeworld";
-	}
-
-	if (obj_creation.custom=0){
-	    if (obj_creation.fleet_type!=1){
-	        flagship_name=obj_creation.flagship_name;
-	        battle_barges=1;
-	        strike_cruisers=6;
-	        gladius=7;
-	        hunters=3;
-			
-			if (global.chapter_name="Soul Drinkers") then gladius-=4; strike_cruisers-=3; battle_barges+=1; 
-			
-			}
-	    if (obj_creation.fleet_type=1){
-			strike_cruisers=8;
-	        gladius=7;
-	        hunters=3;
-		
-	        if (global.chapter_name="Imperial Fists")
-	           {
-				   flagship_name="Spear of Vengeance";
-				   battle_barges+=1;
-			   }
-	    
-		if (global.chapter_name="Crimson Fists")   
```

Comment:
why is this deleted? it's for the new Chapter

Reply:
something must have gone wrong when i pulled the branch back to work on it. fixed.

Reply:
this is still deleted